index.html

```diff
@@ -0,0 +1,92 @@
+<link rel="stylesheet" href="assets/pe-icon-7-stroke/css/pe-icon-7-stroke.css">
+
+</head>
+
+<body>
+
+	<div class="loginCon"> 
+		<div class="row">
+			<!--LeftSide-->
+			<div class="left-log column-large-6 column-medium-5 show-for-large align-center cover-content">
+                <div class="bgimg"></div>
+			</div>
+ 
+            <!--RightSide-->
+            <div class="right-log column-large-6 column-medium-5 column-small-8 align-center cover-content">
+                <div class="whiteLogo show-for-small" alt="white">
+                <img src="images/LogoWhite.svg" alt="logowhite">
+            </div>
+                <div class="vertical-logo align-center show-for-medium">
+                     <img src="images/logo.svg" alt="logo">
+                </div>
+            <div class="loginForm">
+            <form name="login-form" id="login-form" class="contact-form js-animation" method="pos"> 
+            	   <p>Welcome back! Please login to your account.</p>
+
+               <!--  <div class="form-field">
+                    <input id="username" class="input-field text-left" type="text" required>
+                    <span class="lnr lnr-user"></span>
+                    <label class="label" for="username">UserName*</label>
+                </div>
+                <div class="form-field">
+                    <input id="pasword" class="input-field text-left" type="password" required>
+                     <span class="lnr lnr-lock"></span>
+                    <label class="label" for="password">Password*</label>
+                </div> -->
+
+                <div class="form-field-check row text-center align-center">
+                        <div class="checkbox column-large-6">
+                         <input type="checkbox" name="RememberMe" value="Remember"> Remember Me
+                        </div>
+                       
+                        <div class="link column-large-6 show-for-medium">
+                         <p><a href="https://www.w3schools.com/html/">Forgot Password</a></p>
+                        </div>
+                </div>
+
+                <div class="form-field row column-large-12 column-small-6">
+                        <button class="button-white">Login</button>
+                    <button class="button-blue">Login with Facebook</button> 
+                    
+     			</div>
+
+                <div class="link2 show-for-small">
+                    <p><a href="https://www.w3schools.com/html/">I forgot my password!</a></p>
+                      </div>
+                <div class="register row align-center no-margin-bottom text-uppercase">
+
+                <div class="newUser column-small-6 text-right">
+                        <p>New User?</p></div>
+
+                    <div class="register column-small-6 text-left" style="text-decoration: underline;padding-left: 5px;">
+                    <p><a href="https://www.w3schools.com/html/">Register Here</a></p> 
```

Comment:
This text is not aligned with the rest of the content on mobile:
![image](https://user-images.githubusercontent.com/32714864/49344337-c3061e80-f643-11e8-91a4-bc5bd0aee53d.png)

Reply:
fixed